widgets/SensitiveRecordFilter-splittertransform.json

```diff
@@ -199,6 +223,19 @@
           "name": "dlp-tls-enabled"
         }
       ]
+    },
+    {
+      "name": "Filter for simple condition objects - 1",
```

Comment:
what is `Filter for simple condition objects - 1`?

Reply:
We used the same name as it was specified in other widgets when `customTemplateEnabled` was present. 

https://github.com/data-integrations/dlp-plugins/blob/561e837d836800641bd6e21964230b7cf1bce0a1/widgets/SensitiveRecordDecrypt-transform.json#L468-L475

https://github.com/data-integrations/dlp-plugins/blob/561e837d836800641bd6e21964230b7cf1bce0a1/widgets/SensitiveRecordRedaction-transform.json#L724-L731

When `customTemplateEnabled` is NO (disabled),  you can't specify any template. I think that's the reason why it as named like this.

Reply:
I see, thanks